tests/test_store.py

```diff
@@ -93,9 +93,12 @@ def test_get_valid_id_returns_id_and_200(self):
         test_json = json.loads(updated_message)
         expected_id = test_json['tx_id']
 
-        self.app.post(self.endpoints['responses'],
-                      data=updated_message,
-                      content_type='application/json')
+        response = self.app.post(self.endpoints['responses'],
+                                 data=updated_message,
+                                 content_type='application/json')
+
+        self.assertIn(b'data_size_in_bytes', response.data)
```

Comment:
If I'm remembering correctly, these tests are basically integration tests with an in-memory database right?  Could you compute the value of the size + hash, hit the end point, then test to make sure the returned value is what you're expecting?  Just testing that the key is in the response is a pretty weak test